tests/e2e/velero_yaml_suite_test.go

```diff
@@ -0,0 +1,42 @@
+package e2e
+
+import (
+	"path/filepath"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+type veleroTemplateParams struct {
```

Comment:
Better to use dpaTemplateParams or something to make it more generic

Reply:
Done